apps/get_it/lib/app/favourites/favourites_page.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:core/models/app_models/profiles_data.dart';
+import 'package:core/models/tmdb/tmdb_movie_basic.dart';
+import 'package:core/persistence/data_store.dart';
+import 'package:core/ui/movies_grid.dart';
+import 'package:core/ui/scrollable_movies_page_builder.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it_mixin/get_it_mixin.dart';
+
+class FavouritesPage extends StatelessWidget with GetItMixin {
+  @override
+  Widget build(BuildContext context) {
+    final dataStore = get<DataStore>();
+    final profilesData = dataStore.profilesData();
+    return StreamBuilder<ProfilesData>(
+      stream: profilesData,
+      builder: (context, snapshot) {
+        if (snapshot.hasData) {
+          final profilesData = snapshot.data!;
+          final profileFavouriteMovies =
+              dataStore.favouriteMovies(profileId: profilesData.selectedId!);
+          return StreamBuilder<List<TMDBMovieBasic>>(
```

Comment:
This second `StreamBuilder` depends on the value obtained from the first.
Is there a way to register things with `get_it` so that two nested `StreamBuilder`s are not required?